VideoLocker/src/main/java/org/edx/mobile/player/PlayerController.java

```diff
@@ -763,7 +766,9 @@ public void onClick(View v) {
 //          }
         }
     };
-
+    /**
+     * Currently unused 3/4/2015

```

Comment:
I think it was written there with the intention of being hooked up. There is a possibility that we might not even need this feature.